boards/nxp/frdm_k64f/frdm_k64f.dts

```diff
@@ -18,6 +18,7 @@
 		sw1 = &user_button_2;
 		magn0 = &fxos8700;
 		accel0 = &fxos8700;
+		mcuboot-button0 = &user_button_3;
```

Comment:
Just curious, how did you decide which button to use.

Reply:
> Just curious, how did you decide which button to use.

All these boards have at least one button and it is sw0.

Reply:
Chosen the wrong button can cause this feature to not work at all. See e.g. https://github.com/zephyrproject-rtos/zephyr/pull/79715

Were these configurations tested?